src/Abp/Runtime/Caching/IAbpCache.cs

```diff
@@ -0,0 +1,25 @@
+using System.Threading.Tasks;
+
+namespace Abp.Runtime.Caching
+{
+    /// <summary>
+    /// Base interface for caches.
+    /// </summary>
+    public interface IAbpCache
```

Comment:
We get an `IAbpCache `interface. A cache interface but it only has a `Clear` method, which may be confusing. Is it possible to merge it into the `ICache `interface.

I even think we can merge all the methods of the `IAbpCacheBatchActions `and `IAbpCacheActions `interfaces into the `ICache `interface, is it necessary to extract the cache interface that supports single and multiple keys?

This is just my plain thoughts.

Reply:
Agree, i extract cache actions/batch actions base initially was for the custom redis  cache implementation in https://github.com/aspnetboilerplate/aspnetboilerplate/pull/4689.

However, i think we can keep only `IAbpCache` and `IAbpCache<TKey, TValue>` only